src/main/java/com/amalitech/gpuconfigurator/config/JwtAuthenticationFilter.java

```diff
@@ -0,0 +1,86 @@
+package com.amalitech.gpuconfigurator.config;
+
+import com.amalitech.gpuconfigurator.service.JwtService;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.lang.NonNull;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.web.authentication.WebAuthenticationDetailsSource;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+
+/**
```

Comment:
I feel comments for this particular filter is redundant as it is the same configuration throughout all java springboot jwt filter classes